### PR TITLE
Added a `to equal` assertions

### PR DESCRIPTION
**Pull request by sunesimonsen**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -360,6 +367,42 @@ const unexpectedEnzyme = {
...
+            } else {
+              expect.fail();
+            }
+          });
+        }
+      }
+    );
+
+    childExpect.exportAssertion(
+      '<ReactWrapper> [not] to equal <ReactElement>',
```
Comment (sunesimonsen) [inline src/index.js]: I'm getting second thoughs on this name. Maybe we use `to exhaustively satisfy`.

Comment (sunesimonsen) [reply src/index.js]: It would also make sense to satisfy against an array of react elements.

**Comment by sunesimonsen** [comment]
@albertfdp I think I will rework this a bit.

**Comment by sunesimonsen** [comment]
I would like to add the following assertions:

```js
<ReactWrapper> [not] to equal <ReactWrapper>
<ReactWrapper> [not] to exhaustively satisfy <array[ReactElement]>
<ReactWrapper> [not] to satisfy <array[ReactElement]>
```
But I'll just to that in another PR.

**Review by albertfdp — APPROVED** [review]
This is 💯

**Pull request merged**